docs/readmes/faq/faq_magma.md

```diff
@@ -79,9 +79,9 @@ This section lists some of the commonly asked questions related to Magma operati
     - From the popup window, select tab **COMMANDS**. Then, check section **Reboot**.
     - Click on **Reboot** to reboot the AGW node.
 
-### Does AGW supports IPv6?
+### Does AGW support IPv6?
```

Comment:
Nit: intuitively I would say `Does the AGW support IPv6?`

Reply:
Good point, thank you!